internal/pkg/deploy/cloudformation/stack/testdata/workloads/rdws-manifest.yml

```
@@ -50,3 +50,9 @@ environments:
     command: crazy diamond
     variables:
       LOG_LEVEL: debug        # Log level for the "test" environment.
+  prod:
+    network:
+      vpc:
+        placement:
+          subnets: ["mockSubnetARN1", "mockSubnetARN2"]
```

nit: there are no ARNs :) just IDs
```suggestion
          subnets: ["mockSubnetID1", "mockSubnetID2"]
```